### PR TITLE
feat: remind contributors about changelog update via PR template

### PR DESCRIPTION
**Pull request by galargh**
This PR template will be used any time a new PR is opened in Kubo. It should serve as a reminder to the contributor to think whether the change being introduced requires a changelog update or not. It should be treated as a starting point for the effort of making sure we get better with keeping the changelog up to date.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!--
+PR Creation Checklist
+- [ ] Update Changelog
+-->
```
Comment (galargh) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:4]: I put it inside a comment to emphasise it's for contributor to decide what to do with it.

**Comment by ajnavarro** [comment]
This is similar to the approach that we had here: https://github.com/src-d/gitbase/pull/987

From my experience, it helps the PR creator to remember these small things that you tend to forget when adding more code, and also helps the reviewer to check that all needed parts to consider the feature or fix done are there. 

That list cannot be super extensive and it must go to the point, to avoid cumbersomeness when working normally.

**Comment by aschmahmann** [comment]
triage: @galargh is going to take a look this week

**Comment by galargh** [comment]
The PR is ready for review. I updated the description accordingly.

**Review by hacdias — APPROVED** [review]

**Pull request merged**